manage.py

```diff
@@ -45,7 +45,8 @@
 @manager.option("--email", dest="email")
 @manager.option("--user-type", dest="user_type", default="RDU_USER")
 def create_local_user_account(email, user_type):
-    if is_whitelisted_or_government_email(email):
+    form = AddUserForm(email=email, user_type=user_type)
+    if form.email.validate(form):
```

Comment:
Can we not use `form.validate_on_submit()` as we do elsewhere, rather than just validating the one field?  We might add validations to the form in future and would expect them to be called by default.

Reply:
This isn't a request so it isn't ever submitted/POSTed. It's a management command. I've just reproduced the slightly hack-y behaviour of the command. I started doing a bigger refactor of it to clean it up, but decided that would distract from the focus of the PR. Happy to just chuck in some refactoring here as a separate PR on the side. At the end of the day this is really only for devs on local machines, so it might be fine anyway.